### PR TITLE
[Merged by Bors] - Remove `CachedDB.GetEpochWeight`

### PR DESCRIPTION
**Pull request by poszu**
## Motivation

It's not needed

## Description

- Removed `CachedDB.GetEpochWeight()`.
- Removed `TestHandler_processBlockATXs` that essentially tested the  `CachedDB.GetEpochWeight()`, not the ATX Handler.
- Updated the tortoise tests to get epoch weight from the atxsdata store.

## Test Plan

existing tests pass

## TODO

- [x] Explain motivation or link existing issue(s)
- [x] Test changes and document test plan
- [ ] Update documentation as needed
- [ ] Update [changelog](../CHANGELOG.md) as needed

### CODE REVIEW

File: datastore/store_test.go [hunk 1]
```diff
@@ -410,3 +410,50 @@ func TestBlobStore_GetActiveSet(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, codec.MustEncode(as), got)
 }
+
+func TestCachedDb_GetEpochWeight(t *testing.T) {
```
Comment (dshulyak) [inline datastore/store_test.go]: it makes sense to drop this method, i found that it is used only in:
- handler_test.go
- tortoise/model/core.go

in both cases it accidentally is using CacheDb

Comment (dshulyak) [reply datastore/store_test.go]: actually after you removed it in handler_test.go it is used only in that tortoise testing file

Comment (poszu) [reply datastore/store_test.go]: Removed

**Review by dshulyak — APPROVED** [review]

**Comment by poszu** [comment]
bors merge

**Comment by poszu** [comment]
bors merge

**Review by fasmat — APPROVED** [review]

**Pull request closed without merging**